pkg/kongstate/consumer_test.go

```diff
@@ -69,10 +72,13 @@ func TestConsumer_SanitizedCopy(t *testing.T) {
 
 func TestConsumer_SetCredential(t *testing.T) {
 	username := "example"
+	standardVersion, _ := semver.Make("2.3.2")
+	mtlsUnsupportedVersion, _ := semver.Make("1.3.2")
```

Comment:
Similar to above comments, I would recommend we use `MustParse(v)` for this:
```suggestion
	standardVersion := semver.MustParse("2.3.2")
	mtlsUnsupportedVersion := semver.MustParse("1.3.2")
```